suite-native/module-settings/src/slice.ts

```diff
@@ -1,11 +1,12 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
-import { ThemeColorVariant } from '@trezor/theme';
+import { fiatCurrenciesMap, Currency, CurrencyType } from '@suite-common/suite-config';
 
-export type AppColorScheme = ThemeColorVariant | 'system';
+import { AppColorScheme } from './types';
 
 export interface AppSettingsState {
     colorScheme: AppColorScheme;
+    currency: Currency;
 }
 
 type SliceState = {
```

Comment:
please rename this to SliceRootState

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/088ccfb38beef6c123b6dd7a50d6ae4c2f120a07